src/scenes/AcceleratorRouter/Modules/AddEventModal.tsx

```diff
@@ -0,0 +1,312 @@
+        const foundProject = allCohortProjects?.find((pr) => pr.id.toString() === projId.toString());
+        projectsWithCohort.push({ cohortId: ps.cohort.id, projectId: Number(projId), projectName: foundProject?.name });
+      })
+    );
+
+    onSave({ ...record, projects: projectsWithCohort });
+  };
+
+  const getProjectsForCohort = (selectedCohort: CohortModuleWithProject) => {
+    const allProjects: DropdownItem[] = [];
+    selectedCohort?.participants?.forEach((pp) => {
+      pp.projects.forEach((proj) => {
+        allProjects.push({ label: proj.name, value: proj.id });
+      });
+    });
+
+    return allProjects;
+  };
+
+  const updateProjectSectionCohort = (index: number, newCohort: CohortModuleWithProject) => {
+    setProjectsSections((prev) => {
+      const oldProjectSections = [...prev];
+      if (oldProjectSections[index]) {
+        oldProjectSections[index].cohort = newCohort;
+      } else {
+        oldProjectSections[index] = { cohort: newCohort, projectIds: [] };
+      }
+
+      return oldProjectSections;
+    });
+  };
+
+  const onAddProject = (index: number, projectId: string) => {
+    setProjectsSections((prev) => {
+      const oldProjectSections = [...prev];
+      if (oldProjectSections[index]) {
+        oldProjectSections[index].projectIds.push(projectId);
+      }
+
+      return oldProjectSections;
+    });
+  };
+
+  const onRemoveProject = (index: number, projectId: string) => {
+    setProjectsSections((prev) => {
+      const oldProjectSections = [...prev];
+      if (oldProjectSections[index]) {
+        const foundIndex = oldProjectSections[index].projectIds.findIndex((pId) => pId === projectId);
+        oldProjectSections[index].projectIds.splice(foundIndex, 1);
+      }
+
+      return oldProjectSections;
+    });
+  };
+
+  const addProjectsSection = () => {
+    setProjectsSections((prev) => {
+      const oldProjectSections = [...prev];
+      oldProjectSections.push({ cohort: {}, projectIds: [] });
+
+      return oldProjectSections;
+    });
+  };
+
+  const removeProjectsSection = (index: number) => {
+    setProjectsSections((prev) => {
+      const oldProjectSections = [...prev];
+      oldProjectSections.splice(index, 1);
+
+      return oldProjectSections;
+    });
+  };
+
+  return (
+    <DialogBox
+      onClose={onClose}
+      open={true}
+      title={strings.EVENT_DETAILS}
+      size='large'
+      middleButtons={[
+        <Button
+          id='cancel'
+          label={strings.CANCEL}
+          type='passive'
+          onClick={onClose}
+          priority='secondary'
+          key='button-1'
+        />,
+        <Button id='save' onClick={save} label={eventToEdit ? strings.UPDATE : strings.ADD} key='button-2' />,
+      ]}
+      scrolled
+    >
+      <Grid container textAlign={'left'} spacing={2}>
+        <Grid item xs={6} sx={{ marginTop: theme.spacing(2), paddingRight: 1 }}>
```

Comment:
We should check with designs but I think we will need a way to pick timezones. I think the FE may default to browser timezone when selecting/picking a time, which will lead to some inconsistencies.

Reply:
And similarly we probably want a way to show events time in different timezone. Maybe the events tab can have a dropdown for timezone for viewing too but likely a design question to answer

Reply:
Yes, I can add timezone in a follow up PR